client/src/components/MemberProfilePage/MemberProfilePage.js

```diff
@@ -0,0 +1,170 @@
+import React, { Component } from "react";
+import "./MemberProfilePage.css";
+import Navbar from "../Navbar/Navbar";
+import InteractionCard from "../InteractionCard/InteractionCard";
+import PreviousTripsCard from "../PreviousTripsCard/PreviousTripsCard";
+import TripCard from "../TripCard/TripCard";
+import example1 from "../../assets/images/card_small2.png";
+import BookTripCard from "../BookTripCard/BookTripCard";
+import example2 from "../../assets/images/hawaii.png";
+import Connections from "../Connections/Connections";
+import pic1 from "../../assets/images/profile-picture-1.png";
+import pic2 from "../../assets/images/profile-picture-2.png";
+import pic3 from "../../assets/images/profile-picture-3.png";
+import pic4 from "../../assets/images/profile-picture-4.png";
+import Footer from "../Footer/Footer";
+import InputTextField from "../InputTextField/InputTextField";
+import Banner__pink from "../Banner/Banner__pink.png";
+import Banner from "../Banner/Banner";
+import GroupsList from "../GroupsList/GroupsList";
+import Bio from "../Bio/Bio";
+// Class Based React Component
+class MemberProfilePage extends Component {
+  constructor(props) {
+    super(props);
+    console.log(props);
+
+    // Default CSS class to apply to the Component
+    this.state = {
+      classList: "MemberProfilePage"
+    };
+  }
+
+  // Runs after Component is loaded in the broswer
+  componentDidMount() {}
+
+  // Runs after a component has been updated
+  componentDidUpdate() {}
+
+  // Runs right before a component is removed from the DOM
+  componentWillUnmount() {}
```

Comment:
If you don't have any state, then could you turn this into a function and delete all this?